docs/user_guide/emummc/making_emummc.md

```diff
@@ -9,6 +9,7 @@
 
 !!!tip ""
     1. Enter RCM and inject the Hekate payload
+        - If you forgot how to do this, re-read the [sending payload](sending_payload.md) section of the guide.
```

Comment:
Feels redundant. (Includes repetitions later down this PR.)

I don't really see the need for this, we should be able to assume that users who follow our guide linearly maintain the knowledge from previous pages.

Reply:
All the links back to sending_payload.md have been removed from making_emummc.md